configure

```diff
@@ -100,6 +101,40 @@ while [ "$TF_NEED_HDFS" == "" ]; do
   esac
 done
 
+while [ "$TF_NEED_LIBJPEG_TURBO" == "" ]; do
```

Comment:
I'm happy with either solution. The caveat being that we have to make sure that this works on all platforms we (want to) support (incl. Windows and ARM). I agree depending on nasm isn't a problem assuming it's portable. I also agree that it would be nice to avoid an extra configuration question. I wasn't aware we had a complete bazel build. jart@ can you add that file to tensorflow so mkolod@ can use it?